apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,61 @@
-import { Component, OnInit } from '@angular/core';
+import { AsyncPipe } from '@angular/common';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  OnDestroy,
+  OnInit,
+} from '@angular/core';
+import { Subscription } from 'rxjs';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
+import { ListItemComponent } from '../../ui/list-item/list-item.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  templateUrl: './city-card.component.html',
   standalone: true,
-  imports: [],
+  imports: [CardComponent, ListItemComponent, AsyncPipe],
+  styles: [
+    `
+      .bg-light-blue {
+        background-color: rgba(0, 0, 255, 0.1);
+      }
+    `,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class CityCardComponent implements OnInit {
-  constructor() {}
+export class CityCardComponent implements OnInit, OnDestroy {
+  protected cities$ = this.store.cities$;
 
-  ngOnInit(): void {}
+  private _subscriptions = new Subscription();
```

Comment:
I'm not a big fan of subscription, you can use `takeUntilDestroyed`, but that's a personal preference